Add badges to README.md

JacobRoe opened the pull request:
OPTM-90

JacobRoe committed:
Add badges to README.md

OPTM-90

guisedlythe reviewed (changes requested):
We should also add the license to this and and the lww_crdt repository via github (if we choose this license and have checked if we are allowed to apply it with the libraries that we use).

guisedlythe commented on README.md:
```diff
@@ -1,4 +1,7 @@
 # P2P Task Manager
+![CI](https://github.com/PJ-ODS-2021/P2P-Task/actions/workflows/main.yaml/badge.svg)
```
Please add a newline after the section header:
https://github.com/DavidAnson/markdownlint/blob/v0.23.1/doc/Rules.md#md022

JacobRoe committed:
Fix missing newline after header

firstdayofjune commented:
Thanks for your changes. I will check the license compatibility and also look for a way to display the used packages in the settings section.

JacobRoe committed:
Add about section to settings screen

OPTM-87

JacobRoe committed:
Merge branch 'main' into OPTM-90_add-badges-to-readme

# Conflicts:
#	lib/services/network_info_service.dart

Commit:
Checked In-Bound licenses

- checked for each used package what license it uses

refs OPTM-87

Commit:
Added license file

- checked and documented for each used package what license it uses
- added GPL-v3, a strong copy-left license

OPTM-90 #close

firstdayofjune commented:
I added the GPL-v3 license to the project, which makes the app truly free software.
I therefore added everyone as a reviewer to confirm the choice of license.
Refer to https://choosealicense.com/licenses/gpl-3.0/ or https://www.gnu.org/licenses/quick-guide-gplv3.html for more info on the GPL.

JacobRoe committed:
Merge branch 'main' into OPTM-90_add-badges-to-readme
